README.md

```diff
@@ -19,7 +23,8 @@ Extend necessary configs inside `.eslintrc.json`:
   "extends": [
     "@macpaw/eslint-config-base", // required
     "@macpaw/eslint-config-react", // optional
-    "@macpaw/eslint-config-typescript" // optional
+    "@macpaw/eslint-config-typescript", // optional
```

Comment:
Can you pls also update this block in your PR: `@macpaw/eslint-config-base` isn't required since 2.0.0 because all other packages are depend of it under the hood.
Looks like I forgot to update readme when changed that.